xml/gl.xml

```diff
@@ -9177,6 +9179,7 @@ typedef unsigned int GLhandleARB;
             <param><ptype>GLuint</ptype> <name>buffer</name></param>
             <param group="BufferOffset"><ptype>GLintptr</ptype> <name>offset</name></param>
             <param group="BufferSize"><ptype>GLsizeiptr</ptype> <name>size</name></param>
+            <glx type="render" opcode="355"/>
```

Comment:
As an example of a bad opcode here, this appears to assign render opcode 355 to glBindBufferBaseEXT, but that opcode is associated with glBindBufferRange() in our driver, and also in OpenGL3_0.txt in the old subversion tree and in a proposed update to NV_transform_feedback.txt I still have pending in NVIDIA's internal registry mirror, but can't find the Khronos bug for anymore.

Reply:
Yeah, I think the github UI is lying to me.  When I click the little expand button, it makes it look like this is affecting BindBufferBase, but I see the line numbers jump about 30 lines and expand only gave me an additional line or two of context.  When I view the raw file it looks right.  Let me download and review properly in a terminal.